src/sentry/models/dashboard_permissions.py

```diff
@@ -3,10 +3,23 @@
 from django.db import models
 
 from sentry.backup.scopes import RelocationScope
-from sentry.db.models import Model, region_silo_model
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_model
 from sentry.db.models.base import sane_repr
 
 
+@region_silo_model
+class DashboardPermissionsTeam(Model):
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
Should this be `RelocationScope.Organization`? Without that dashboards will 'break' when exported and imported as permissions won't be perserved.